api/raml/support-logging.raml

```diff
@@ -67,10 +69,8 @@ schemas:
         example: 10
     get:
       responses:
-        413: 
-          description: if the number of events exceeds the current max limit
-        503: 
-          description: for unknown or unanticipated issues.
+        400: 

```

Comment:
You'll want to keep the 413 here. It's indicating that there is more data to return than what the client asked for. Since that's determined on the server and not specifically by the request, 413 is appropriate. However the DB query should cap the number of records returned, so I don't think this will be frequent.